spec/abci/abci++_app_requirements.md

```diff
@@ -45,7 +45,7 @@ transaction list returned by the application will never cause the resulting bloc
 limit.
 
 * Requirement 3 [`PrepareProposal`, `ProcessProposal`, coherence]: For any two correct processes *p* and *q*,
-  if *q*'s Tendermint calls `RequestProcessProposal` on *u<sub>p</sub>*,
+  if *q*'s Tendermint calls `ProcessProposal` on *u<sub>p</sub>*,
```

Comment:
Did you decide to change this at a spec level? Think that so far the notation `RequestMethodName` and `ResponseMethodName` was used to differentiate between issuing a request message and receiving the response (as indicated in the description of the methods).

Reply:
Keeping the same format all over the spec is important, but I particularly do prefer this notation. We call `ProcessProposal` (RPC method) by sending a `RequestProcessProposal` message via ABCI (RPC internals).

Reply:
My proposal here was in the sense that  "calls" is normally associated with the method name rather than the parameter. Thanks for the comments I understand what you intended with this usage. I will review the wording before turning this draft into an actual PR.